vars/runTest.groovy

```diff
@@ -137,11 +137,14 @@ Map call(Map config = [:]) {
         }
     }
 
-    int runTime = durationSeconds(startDate)
+    Date endDate = new Date()
+    int runTime = durationSeconds(startDate, endDate)
 
     // We need to pass the rc to the post step.
     Map results = ['result_code': rc,
                    'result': status,
+                   'start_date': startDate,
+                   'end_date': endDate,
```

Comment:
Just curious, how are the actual start and end dates being used vs. the run-time?

Reply:
If you compare the actual start time of of a functional test with the last actual end time for the last unit test to complete, it will tell you how long the functional test was waiting for a cluster to run on.   That calculation is done by the software processing the resulting files.